pkg/cniipwrapper/ip.go

```diff
@@ -47,11 +47,8 @@ func (*cniIP) SetHWAddrByIP(ifName string, ip4 net.IP, ip6 net.IP) error {
 	return ip.SetHWAddrByIP(ifName, ip4, ip6)
 }
 
-func (*cniIP) DelLinkByNameAddr(ifName string) (*net.IPNet, error) {
+func (*cniIP) DelLinkByNameAddr(ifName string) error {
```

Comment:
why is the ip return being deleted from the func signature? is it not relevant?

Reply:
nvm I see it in the description under e2e tests, another nitpick but maybe put this bit in the "summary" and "Description for changelog" since this is the main purpose of this PR, IIUC

Reply:
Yeah we don't use it. 
https://github.com/aws/amazon-ecs-cni-plugins/blob/a04bd6ecae16f6b2bc2957e3cafbf365c0710b19/plugins/ecs-bridge/engine/deletelink.go#L41

Reply:
> nvm I see it in the description under e2e tests, another nitpick but maybe put this bit in the "summary" and "Description for changelog" since this is the main purpose of this PR, IIUC

Np will do!